.editorconfig

```diff
@@ -1,3 +1,20 @@
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.rs]
+indent_size = 4
+indent_style = space
+
+[*.toml]
+indent_size = 2
+indent_style = space
+
 [*.scm]
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
https://github.com/tree-sitter/tree-sitter/blob/master/cli/src/templates/.editorconfig#L14

Reply:
I only set it to `4` because that’s what was already there. I’ll always prefer `2`.

Reply:
I think it would be better to follow the tree-sitter configuration here. It makes copying&pasting easier, hehe.